ops/services/alerts/app_service_metrics/function_triggers.tf

```diff
@@ -0,0 +1,92 @@
+resource "azurerm_monitor_alert_processing_rule_suppression" "function_triggers_suppression" {
+  name                = "${var.env}-function-triggers-suppression"
+  resource_group_name = var.rg_name
+  scopes              = [data.azurerm_resource_group.app.id]
+
+  condition {
+    target_resource_type {
+      operator = "Equals"
+      values   = ["Microsoft.Web/azureFunctions"]
+    }
+    severity {
+      operator = "Equals"
+      values   = ["Sev4"]
+    }
+  }
+
+  schedule {
+    effective_from  = "2022-01-01T01:02:03"
+    effective_until = "2022-02-02T01:02:03"
+    time_zone       = "Eastern Standard Time"
+    recurrence {
+      daily {
+        start_time = "20:00:00"
+        end_time   = "09:00:00"
+      }
+      weekly {
+        days_of_week = ["Saturday", "Sunday"]
+      }
+    }
+  }
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "batched_uploader_function_not_triggering" {
+  name                = "${var.env}-batched-uploader-function-not-triggering"
+  description         = "QueueBatchedReportStreamUploader is not triggering on schedule"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+  severity            = 4
```

Comment:
`Question, clarity, not blocking:` 
It looks like we set all our other alerts at `severity = 3`, or we pass it from the module instantiation via the severity variable. In my quick check, we only seem to use the variable in prod where we set it to `1`. These numbers seem _mostly_ bogus because we manage our alerts in pagerduty via the action groups. As far as I can tell, a `1`,  `2`,  and `3` are treated all the same depending on which action group they are in (`prod`, `non-prod`).  `4` is now a bit more special because of the `rule_suppression`. Is that about right?

Reply:
That's correct! Since we weren't using anything with `4`, I opted to assign that as our selector for the suppression.